release-tools/MKRELEASE.md

```diff
@@ -3,76 +3,87 @@
 This file provides an overview of the mkrelease.pl script, and how to
 configure some of its parameters (by setting environment variables).
 It is normally run by following the process in README.md and should be
-run in a pristine directgory of the branch to be released, which must
+run in a pristine worktree of the branch to be released, which must
 be a stable branch.
 
-The script handles most of the processes involved in making a release
+The script handles most of the processes involved in making a release,
 including:
 
 1. Doing `make update`
 2. Changing version numbers in various files
-3. Tagging release
+3. Tagging the release
 4. Making the tarballs, .md5, .sha1, .sha256 and .gpg files
-5. Creating the signed email announcement, including hashes of release
+5. Creating the signed email announcement, including hashes of the release
 6. Uploading files to dev.openssl.org
-7. Updating the version for next release
+7. Updating the version for the next release
 
 If you plan to make an actual release make sure your GPG key is included
 in the [OMC list](https://www.openssl.org/community/omc.html) on the website
 and in the file doc/fingerprints.txt of *all* active branches.
 
-The script mkrelease.pl calls the files release-check.pl, release-date.pl,
-release-git.pl, release-update.pl and release-version.pl, which are expected
-to all be in the same directory.
+The script `mkrelease.pl` calls the files `release-check.pl`, `release-date.pl`,
+`release-git.pl`, `release-update.pl` and `release-version.pl`, which are expected
+to all be in the release-tools directory. In the following, we will assume that the
+release-tools directory was added to the PATH variable as described in the
+[README](README.md#prepare-the-path-variable) file.
 
-You are advised to clone a fresh repository. Unless you specify `--no-clean`
-the release process will run `git clean -x -d -f`
-*WARNING* this will delete ALL untracked files from the current branch.
+*WARNING*  You are advised to run this command only in a fresh worktree which
+contains no uncommitted changes or untracked files, because unless you specify
+`--no-clean`, the release process will run `git clean -x -d -f`, which will
+delete ALL untracked files from the worktree.
+See the [README](README.md#prepare-the-worktrees) file for a suggestion how to
+prepare the worktrees.
 
-From the branch directory if you run:
+Change into the worktree and run:
 
-        perl /path/to/mkrelease.pl --reviewer=name
+        mkrelease.pl --reviewer=name
```

Comment:
Elsewhere, we're sticking to 4 space indent for verbatim stuff...  How about here (and I do mean throughout the file)?